pegasus/sites.v3/code.org/public/ops-dashboard/index.haml

```diff
@@ -501,15 +502,22 @@ title: "Ops Dashboard"
         if (teacher.id) {
           teacher.editing = false;
         } else {
-          $scope.cohort.teachers.splice($scope.cohort.teachers.indexOf(teacher), 1); // remove from array
+          if (teacher.newTeacher) {
```

Comment:
oh actually I guess you don't actually need to do this if else because new teachers are always in newTeachers and are never in teachers